pkgs/tools/backup/rdiff-backup/default.nix

```
@@ -1,36 +1,34 @@
-{ lib, python3Packages, fetchPypi, librsync }:
-
-let
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+  librsync,
+}: let
   pypkgs = python3Packages;
-
 in
-pypkgs.buildPythonApplication rec {
-  pname = "rdiff-backup";
-  version = "2.0.5";
+  pypkgs.buildPythonApplication rec {
+    pname = "rdiff-backup";
+    version = "2.2.5";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "sha256-VNFgOOYgFO2RbHHIMDsH0vphpqaAOMoYn8LTFTSw84s=";
-  };
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "sha256-huKCa3hOw+pO8YfZNu5fFSd0IsQHfvoBVu9n4xOeoI4=";
+    };
 
-  # pkg_resources fails to find the version and then falls back to "DEV"
-  postPatch = ''
-    substituteInPlace src/rdiff_backup/Globals.py \
-      --replace 'version = "DEV"' 'version = "${version}"'
-  '';
+    buildInputs = [librsync];
 
-  buildInputs = [ librsync ];
+    nativeBuildInputs = with pypkgs; [setuptools-scm pyyaml];
```

`pyyaml` is only needed in `propagatedBuildInputs`.
```suggestion
    nativeBuildInputs = with pypkgs; [setuptools-scm];
```

---

That makes sense. Reverted to the original formatting, removed `librsync` from `buildInputs` and `pyyaml` from `nativeBuildInputs`.

Also realized I have to really learn how to properly make changes to a running pull request. That whole force pushing saga is pretty embarrassing.